apiary.apib

```diff
@@ -2618,6 +2618,23 @@ Get a list of products.
                         + description: `dark chocolate` (string)
                 + external_id: `123` (string)
 
+### products.add [POST /products.add]
+
+ Add a new product.
+
+ + Request (application/json;charset=utf-8)
+
+     + Attributes (object)
+        + name: `cookies` (string, required)
```

Comment:
```suggestion
        + name: `cookies` (string, optional)
```

Both name and code should be optional. But 1 of them should be filled in. How can we make that clear in the documentation?